src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/SetAzureRmVmssRollingUpgradePolicyCommand.cs

```diff
@@ -74,6 +74,12 @@ public partial class SetAzureRmVmssRollingUpgradePolicyCommand : Microsoft.Azure
             ValueFromPipelineByPropertyName = true)]
         public bool PrioritizeUnhealthyInstance { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        public bool MaxSurge { get; set; }
```

Comment:
Please add help message for this parameter and refresh help docs for this command

Reply:
added, thanks.

Reply:
This parameter should be nullable bool. It needs a state to describe no change.

Reply:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1293#issuecomment-1349246857

Here is the explanation provided by the author for keeping this boolean.